### PR TITLE
DEV-50: Post-build script tweaks

### PR DESCRIPTION
**Pull request by jherico**
* [DEV-59](https://highfidelity.atlassian.net/browse/DEV-59) Remove Server Console (and nitpick)
* [DEV-60](https://highfidelity.atlassian.net/browse/DEV-60) Remove jsdoc
* [DEV-61](https://highfidelity.atlassian.net/browse/DEV-61) Remove serverless content
* [DEV-67](https://highfidelity.atlassian.net/browse/DEV-67) Create different output filenames for Mac and PC
* [DEV-66](https://highfidelity.atlassian.net/browse/DEV-66) Remove top level folder in zip archive

Mac ZIP archive is now ~180 MB compared to 370 for the dmg
Win ZIP archive is now ~190 MB compared to 276 for the exe installer

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19678/HighFidelity-Beta-PR15640-dd67940.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19678/HighFidelity-Beta-Interface-PR15640-dd67940.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19678/HighFidelity-Beta-PR15640-dd67940.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19678/HighFidelity-Beta-PR15640-dd67940.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19678/HighFidelity-Quest-Beta-PR15640-dd67940.apk)

File: tools/ci-scripts/postbuild.py [hunk 1]
```diff
@@ -1,20 +1,59 @@
...
-#    print("{} = {}".format(var, os.environ[var]))
+    if RELEASE_TYPE == "PRODUCTION":
+        BUILD_VERSION = "{}-{}".format(RELEASE_NUMBER, GIT_PR_COMMIT_SHORT)
+    elif RELEASE_TYPE == "PR":
+        BUILD_VERSION = "PR{}-{}".format(RELEASE_NUMBER, GIT_PR_COMMIT_SHORT)
+    else:
+        BUILD_VERSION = "dev"
+
+    if sys.platform == "win32":
+        PLATFORM = "win"
```
Comment (mattr1x) [inline tools/ci-scripts/postbuild.py]: The rest of the pipeline expected this to be windows.

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19681/HighFidelity-Beta-PR15640-aba6455.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19681/HighFidelity-Quest-Beta-PR15640-aba6455.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19681/HighFidelity-Beta-PR15640-aba6455.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19681/HighFidelity-Beta-Interface-PR15640-aba6455.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19683/HighFidelity-Beta-PR15640-1ab2507.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19683/HighFidelity-Beta-Interface-PR15640-1ab2507.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19683/HighFidelity-Beta-PR15640-1ab2507.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19683/HighFidelity-Beta-Interface-PR15640-1ab2507.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19683/HighFidelity-Beta-PR15640-1ab2507.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19683/HighFidelity-Quest-Beta-PR15640-1ab2507.apk)

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19684/HighFidelity-Beta-PR15640-4d0eb65.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19684/HighFidelity-Quest-Beta-PR15640-4d0eb65.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19684/HighFidelity-Beta-PR15640-4d0eb65.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19684/HighFidelity-Beta-Interface-PR15640-4d0eb65.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19684/HighFidelity-Beta-PR15640-4d0eb65.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19684/HighFidelity-Beta-Interface-PR15640-4d0eb65.exe

**Comment by jherico** [comment]
Current outputs:

https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19684/HighFidelity-Beta-Interface-PR15640-4d0eb65-windows.zip

https://s3.amazonaws.com/hifi-deployment/jobs/pr-build/label%3Dmacos/19684/HighFidelity-Beta-Interface-PR15640-4d0eb65-mac.zip

**Pull request merged**